lib/internal/webstreams/adapters.js

```diff
@@ -386,10 +390,16 @@ function newReadableStreamFromStreamReadable(streamReadable) {
   // back to a minimal strategy that just specifies the highWaterMark
   // and no size algorithm. Using a ByteLengthQueuingStrategy here
   // is unnecessary.
-  const strategy =
-    objectMode ?
-      new CountQueuingStrategy({ highWaterMark }) :
-      { highWaterMark };
+
+  const evaluateStrategyOrFallback = (strategy) => {
```

Comment:
I would suggest:
```js
  const evaluateStrategyOrFallback = (strategy) => {
    // If there is a strategy available, use it
    if (strategy)
      return strategy;

    // Fallback to to default behavior
    //
    // When not running in objectMode explicitly, we just fall
    // back to a minimal strategy that just specifies the highWaterMark
    // and no size algorithm. Using a ByteLengthQueuingStrategy here
    // is unnecessary.
    if (objectMode) {
      return new CountQueuingStrategy({ highWaterMark });
    } else {
      return { highWaterMark };
    }
  };
  ```

Reply:
Makes sense to me although I'm not really a fan of ``if-else`` branching

Reply:
Thanks for the nit though!

Reply:
It's great! Thanks for your contribution.